docs/pull_request_template.md

```diff
@@ -1,17 +1,31 @@
-Fixes P4-<TODO>
+### Jira link
 
-## Proposed Changes
+P4-<TODO>
 
-- TODO
+### What?
 
-## To test/demo change
+I have added/removed/altered:
 
-- TODO
+- [ ] Added foo in bar
+- [ ] Removed baz
 
-## Things to check and link
-- [ ] API docs has been updated if necessary
-- [ ] New environment variables have been added to staging configuration
+### Why?
 
-## Manual steps to deploy/dependencies
+I am doing this because:
 
-- TODO (or remove)
+-
+-
+-
+
+### Acceptance Criteria
```

Comment:
Does the AC should stay in the Jira ticket?

Reply:
Yes, lets not replicate stuff from Jira here